### ISSUE TITLE
Found another issue related this problem :(

### ISSUE THREAD

**Issue by Chucqui** [freeCodeCamp/freeCodeCamp#13794]
Challenge [Escape Sequences in Strings](https://www.freecodecamp.com/challenges/escape-sequences-in-strings#?

My code: i think the right one would be== "FirstLine\n\\\Secondline\\\\rThirdLine"; 
but with this code i get 3 error messages back, but when i use the "carrage return" before the "backslach escape"like this,
"FirstLine\n\\\Secondline\r\\\ThirdLine";
 i only get one error message,
(myStr should have encoded text with the proper escape sequences and no spacing).

Then i went with this issue over to;  https://github.com/freeCodeCamp/freeCodeCamp/issues/13652
and try to post my work over ther and i found that the output of my code is not the same in display after hittin the "comment" button.
The first image is before hitting the button 

![ithinkifoundaotherbug](https://cloud.githubusercontent.com/assets/16696230/23664833/03674b52-0357-11e7-909b-537c2f216791.PNG)

![andyessthisisit](https://cloud.githubusercontent.com/assets/16696230/23664865/1c73e038-0357-11e7-80ad-b34f96e849a9.PNG)

and the other after...

I hope you guys guet this, keep it up!!

**Comment by Coby-T** [comment]
The issue you are running into posting has to do with markdown. It is markup language like HTML. When posting to GitHub, a few characters need escape sequences as well. Namely in your case backspaces should be written `\\` write it within a code block within \`.

**Comment by texas2010** [comment]
@Chucqui please fix your message. it is really mess. i cant read it as well. thanks

**Comment by snak3codes** [comment]
Please explain yourself more.

**Comment by cdrainxv** [comment]
@Chucqui: This is not a bug. You have errors in your code:
1. Notice the distinction between `Secondline` and `SecondLine`.
2. The backslash escape `\\` must be on the same line as `SecondLine`

Note: This page is for only reporting issues. You can seek help and/or guidance for these and other challenges in the chat rooms fcc provides such as the [Help JavaScript](https://gitter.im/FreeCodeCamp/HelpJavaScript) room.

**Comment by Chucqui** [comment]
Well about the output on the screen when i hit "COMMENT" (the green little box on the topdown right corner of the box you are reading right now... Yeah! That one!!), I get it, but the code still doesnt pass with the square before, after, on the top, you choose
https://github.com/cdrainxv: .
Please understand that englisch is only the 4th language i speak, so i'm learning to code and also "english" https://github.com/texas2010,..
I explain myself,  this is the output it's asking;
FirstLine
\SecondLine\
ThirdLine
How I understand is that= 
 \n = a "new line" or "next line". 
\\ = 1 "\"  and i need to keep one of them on both side's of the string "SecondLine".
\r = "carriage raturn" wich i think it means "coming back to the start of the next line".
so after the string "FirstLine" i put a "\n" so that it goes tho the "next or new" line like this = FirstLine\n",
then with the second one (NOT before leaving the first one in the line above) i start on the next line but i need to escape a "backslash" Before i get to the string itself so i do like this= "\\SecondLine", and then once i finish with the "string" i need to escape another "backslach" and this i think it should be like this= "SecondLine\\, now comes the time that i need to do a ""carriere return" before i even can start with the next "string" wich i do like this= "\rThirdLine, so i take the third one with me to the third line, and then they are also asking to NOT use any space between, wich gives = "FirstLine\n\\SecondLine\\\rThirdLine" 

Please explain me what i am doing wrong, i  need to be shure that i am getting this right.
Here is another screenshot of todays test [
![notgivingup](https://cloud.githubusercontent.com/assets/16696230/23709521/05431eec-041a-11e7-8434-76d7dc82d2bf.PNG)
](url)

And then this, look at the lines i put in there and compare it with they results. I need to have some sleep i [think.](url
![lol](https://cloud.githubusercontent.com/assets/16696230/23710111/e0c10bf4-041b-11e7-9b56-59eb62f779cb.PNG)
)

As explanation is this the best i can do right now, so if you have the code that makes you pass the issue i would be happy to hear from you.
Thanks for your time guys..
Cheers.

**Comment by Chucqui** [comment]
Only screenshots of my code after now... Pfff..[
![screenoutput](https://cloud.githubusercontent.com/assets/16696230/23710431/07a841e6-041d-11e7-9344-f0fc28e827d8.PNG)
](url)

**Comment by cdrainxv** [comment]
@Chucqui:
1. `\\` is the  backslash escape not `\\\` as you have in your new posts.
2. **SecondLine** not Secondline. It's **L** … CAPITAL LETTER.

Once again, this is not so much an issue as it is incorrect code. Please consider seeking help in the [Help JavaScript](https://gitter.im/FreeCodeCamp/HelpJavaScript) chat room for these kind of challenges as you will get a faster response there. Happy coding!

**Comment by Chucqui** [comment]
https://github.com/cdrainxv
Wouw a CAPITAL L is not the same al a l ??? i didn't knew that... lol.
Just say there is a bug and thats it, i reported it because i wanted to help, no more, but you start playing with capital letter while you know that's not the point, look at the screenshots, thats where the problem is. This is the last time you guys hear from me, beeing polite is not a crime you know, i'm a grown man and i know when i'm wrong and it is not the point right now. Is the way that i explain it in my last comment the way to go?? YES IT IS, but you come here to say uppercase this or that, thats why i didnt want to kom with my other issue. Because there always someone like you at the other side, waiting wanting to prove the others wrong and so on. The point was "there is a bug" and you are like no there is no bug,  give me the right code to pas the issue if you know it so well...

**Comment by Marmiz** [comment]
@Chucqui I'm sorry to hear that you're having difficulties with a challenge; if it could be of any consolation escaping character is always a pain, no matter how many years of experience you have 😄 .

But please also bear with us, we are all here devoting our free time trying to help, so no need to be defensive towards other campers. We're your friends here, not foes.

----

That said I'm afraid that this is not a bug, but is your code to be incorrect.
As @cdrainxv suggested, you could find better support in one of the [official chat rooms](http://forum.freecodecamp.com/t/free-code-camp-official-chat-rooms/19390).

We have some chatrooms for other languages besides english as well, maybe that can interest you since English is not your primary language.

----

Now sine you seems a bit frustrated with this challenge I'll try to break it down for you hoping that it will help; but I'm afraid that none here will give you the correct challenge answer since it's not really helpful for your learning process.

Before starting please keep in mind that this challenge looks for the **exact** text, so you have to mind capitalization:

```
secondline  --> incorrect

secondLine  --> correct
```
This means extra focus when typing every single peace of the string. In some screenshot you've sent, some words have the wrong capitalization as @cdrainxv pointed out, therefore your challenge solution won't be accepted.

The key to tackle this challenge is by "converting" this string into the desired code:

**FirstLine`newlinebackslash`SecondLine`backslashcarriage-return`ThirdLine**


So: 
 - we've already talked about the importance of capitalization.
 - you know that in order to create a new line you need `\n`.
 - you know that in order to visualize a single `\` (backslash) you need to type two of those: `\\`.
 - you know that to make a carriage return you have to type `\r`.

 - also don't forget to include your string inside `""` and close it with a `;` like so:
    `var myStr = "[my awesome string goes inside here]";`

----

Good luck with your challenge and happy coding 👍

**Comment by cdrainxv** [comment]
@Chucqui: I want to apologize if you feel that I have slighted you in any way, as that was not my intention. I believe the bug you are referencing is that the console doesn't log out the code in the same format as shown here:
```
FirstLine
\SecondLine\
ThirdLine
```
but just display the variable myStr as `var myStr = ".........................................";`

However, to log myStr to the console you would have to include `console.log(myStr);` in your editor, but even that won't display it as expected because 1) The console on your page is quite small in height, therefore I suggest using the developer console on your browser to view the output of your code (the shortcut keys are `Ctrl + Shift + J` on windows systems) and 2) you have slight errors in your code as @Marmiz pointed out.

Even doing both of the above your output in the developer console will be:
```
FirstLine
\SecondLine\ThirdLine
```
which will pass the challenge, but will not match the expected:
```
FirstLine
\SecondLine\
ThirdLine
``` 
as pointed out in issue #13652. In that same issue it was discussed what the underlying problem is as well as the fact that the challenge was rewritten in the beta: [updated escape challenge](https://beta.freecodecamp.com/en/challenges/basic-javascript/escape-sequences-in-strings).

I am sorry if it seemed like I didn't understand the issue at hand and you deemed me as being impolite and inattentive to your concerns, but this current issue has been acknowledged in the issue you referenced already and the only thing keeping you from moving on to the next challenge is that your current code as shown in the screenshots doesn't pass the tests written for this challenge.

Feel free to post issues as you find them, I won't comment on them if that's your wish, but any issues you find that you think will help elevate the learning, understanding, and/or experience of fellow campers at this site would be greatly appreciated by the whole community. 

Thanks!

**Comment by texas2010** [comment]
someone did fix this challenge.  plus this challenge is changed. you can check on beta.

have a nice day.   

you could try this challenge if you want to. http://beta.freecodecamp.com/challenges/basic-javascript/escape-sequences-in-strings

**Comment by Chucqui** [comment]
https://github.com/Marmiz Thanks for the time you took, it helped me allot! 
https://github.com/cdrainxv Dont need to apoligize yourself for my mistakes, I'm here for like a week and its becoming really addictive, i took some sleep and when i came back my first taught where about how stupid i was for reacting that way and am feeling kind of bad, you should keep up the good work and spread your knowledge with those in need, me i will try to take it more easy. 
I understand completely that you guys are doing this in your free time and as myself i know the feeling that comes with helping others but there are some kind of folks out there that enjoy taking misadventage of their knowledge, by making other feel stupid i mean, while there are others that are just trying to help, Because of the overdossis of coding lessons i just toke i wasnt able to think clearly anymore and fool myself into thinking that you where the first kind of them, my mistake, you just keep beeing that second kind of them. 
Me i need to start putting into my brain that this is not like the other challenges i took before in my life, i am not make this one on my own  and it's maybe the right time to forget the first kind of people i mentioned just before and be happy that i came into one like you from the first time i asked for help in like a month of struggle in my way up to a new career. I found the site in january and when loged in with the first challenge got scared and runned away, i didn't understand a clue about what was going on, i think it was the Github part that made me scared... Then i ran into cs50, it was even worst!! Will never forget my raction then, "learning chinees woud be easear!" i said to myself . That guy is a beast!! Wanted almost to quit but then remembered that joy is in the struggle we take, not in the archivement, and started to read about it and watch some youtube's, came back past week and that's when the magic happened, that's why i was thinking there was some kind of error somewhere and that i could help, because i understand if i am wrong that you say so, but if the machine print's out another code than the one i was putting, for me i see that as an error, if the machine let me pass with different codes that prints out different results of what they are asking for and that just in one assingment, it must be some kind of bug! error! something like that i dont know.
Lets say a teacher asked me '5 + 5' how mutch is it, and they put '3' choices in front of me '9, 10, 11', me i say something like '9', they will "say" no you are wrong it is '10' but at the "same" time they are "showing" me the value of '11'. I think it is my task to point the right value to them because they dont realize the mistake. That is what i was trying to do, help the teacher, but if the teacher keeps saying that the right value of then is still '11' i would be forced to search for other teachers and ask the same question, and if they point out the value of then as '10', me i will soon been be searching for another teacher, wouldn't you?.
I really can't get into my head that it is logic that if you type something into a machine you see something different on the screen there is nothing wrong going on and yess i make mistakes, allot of them!! It make wise but this.. i dont get it!! i knock and knock and still dont get it.. How am i suposed to belief that if the output is diffrent to the input, the machine is going to accept the right answer anyway, like in the last pic of the test that i posted i know it was not the right answer thats why i typed my explenation of my understanding of the question and i think i was right or wasnt i?
All this is me pointing out the teacher because look here's the thing, i am a paranoia, you can call it that way if you like it but today i came into a total different on understandeble sitution. 
What if i told you that there are some questions out there that you can even pass with different lines of codes, with different syntax, that all return another value than the one who is asked, and yess you pass with each one them, ( different lines of code with different syntax and that all returns another value than the one who's asked, you pass guys,) is that what logical means?? 
Beeg up guis end eguein, Thenks fur de lesso'ns, me tipin  much faster all ot dhan wen i timed 1s't the sending or issue mms, end y englich do a beter dhen be4.
And i passed the issue by the way... 👍
